targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/TARGET_ADAFRUIT_NRF52840_FEATHER/USBConsole.h

```diff
@@ -0,0 +1,7 @@
+#include "USBSerial.h"
+
+FileHandle* mbed::mbed_override_console(int)
+{
+    static USBSerial  usbSerialConsole;
+    return &usbSerialConsole;
+}
```

Comment:
I don't see this in the code for [the other MCU_NRF53840 targets](https://github.com/ARMmbed/mbed-os/tree/master/targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840), do we need to override console for this particular targets?